patches/tModLoader/Terraria/RecipeGroup.cs.patch

```diff
@@ -1,24 +1,33 @@
 --- src/Terraria/Terraria/RecipeGroup.cs
 +++ src/tModLoader/Terraria/RecipeGroup.cs
-@@ -8,6 +_,7 @@
+@@ -5,13 +_,15 @@
+ {
+ 	public class RecipeGroup
```

Comment:
Better to have the new members moved to RecipeGroup.TML.cs, but you can do this later.

Reply:
not enough in this case, happy for them to be here.